game/game.hh

```diff
@@ -107,6 +110,7 @@ private:
 	AnimValue m_dialogTimeOut;
 	// Dialog members
 	std::unique_ptr<Dialog> m_dialog;
+	TranslationEngine& m_translationEngine;
```

Comment:
shoudl/could it be const?

Reply:
When making it const i'm getting:
```
Error (active)	E1086	the object has type qualifiers that are not compatible with the member function "TranslationEngine::setLanguage" C:\Users\Arjan\source\repos\performous\performous2\game\game.hh#79	
```

Reply:
Cannot be const due to `std::string getCurrentLanguage() { return m_translationEngine.getCurrentLanguage().second; }`

Reply:
Returns a copy THUS can be const